source/_integrations/lamarzocco.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: La Marzocco
+description: Instructions on how to integrate your La Marzocco coffee machine with Home Assistant.
+ha_release: 2023.10
+ha_category:
+  - Water Heater
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_domain: lamarzocco
+ha_platforms:
+  - water_heater
+ha_codeowners:
+  - '@zweckj'
+  - '@rccoleman'
+ha_integration_type: integration
+---
+
+This integration interacts with [La Marzocco coffee machines]() through a mix of cloud calls, local API calls, WebSocket connections and (optionally) Bluetooth. 
```

Comment:
Did you have a specific, external link in mind here?

Reply:
yep, added, thanks.